CONTACT.rst

```diff
@@ -2,8 +2,13 @@
 Contact
 =======
 
-Do you have question about this project? Please contact us on
-cn-ws@omgeving.vlaanderen.be
+For technical questions, we refer to the documentation and to the frequently asked
+questions in the documentation. If you have a technical issue with running the
+model, or if you encounter a bug, please use the issue-tracker on github:
+https://github.com/watem-sedem/watem-sedem/issues.
 
-For technical questions, we refer to the frequently asked questions in the
-documentation or to the issue-tracker on git.
\ No newline at end of file
+If you have questions about the history or concept of the model that are not answered in
```

Comment:
this should have a reference no? or is this being discussed with Gert, if not: ask him how to contact them

Reply:
yes this contact point shall be added when it is created by Gert. For now we leave it like this.